### PR TITLE
Add cultural build tribute and redeemable views

### PR DESCRIPTION
**Pull request by marthendalnunes**
This PR adds visualizations of the cultural build tribute and the redeemable amount in form of area plots. It also adds a table for better visualization of the chart info, like [this](https://docs.google.com/spreadsheets/d/17IQpL_LddNCW7ELNzySErh8yUFvHj3Xgf_iKa2QhWLA/edit?usp=sharing).

### CODE REVIEW

File: tech/tech.py [hunk 1]
```diff
@@ -89,43 +90,45 @@ def impact_hours_accumulation(self):
...
+        #return pn.Column(historic_curve * historic_bar * predicted_curve * predicted_bar * optimistic_curve * optimistic_bar, f"<b>Predicted Impact Hours: </b>{round(self.total_impact_hours)}")
+        return pn.Column(historic_curve * historic_bar * predicted_curve * predicted_bar * optimistic_curve * optimistic_bar)
+
 
-    
 class ImpactHoursFormula(param.Parameterized):
     """
     Sem's Formula 🌱 🐝 🍯
-    This formala was a collaboration of Sem and Griff for the TEC hatch impact hours formula. 
+    This formala was a collaboration of Sem and Griff for the TEC hatch impact hours formula.
```
Comment (mendesfabio) [inline tech/tech.py:103]:
```suggestion
    This formula was a collaboration of Sem and Griff for the TEC hatch impact hours formula.
```

File: tech/tech.py [hunk 2]
```diff
@@ -163,92 +166,134 @@ def impact_hours_rewards(self):
...
+            impact_hour_rate = R* (self.minimum_raise / (self.minimum_raise + m*H))
+            cultural_build_tribute = (H * impact_hour_rate)/self.minimum_raise
+            df_hatch_params = df_hatch_params.append({'Total XDAI Raised': self.minimum_raise, 'Impact Hour Rate':impact_hour_rate, 'Cultural Build Tribute':cultural_build_tribute, 'Hatch tribute':hatch_tribute, 'Redeemable':hatch_tribute/(1 + cultural_build_tribute), 'label':'Min Raise'}, ignore_index=True)
+            df_hatch_params = df_hatch_params.sort_values(['Total XDAI Raised'])
+
+        df_min_raise = df_hatch_params.query("label == 'Min Raise'")
+        if len(df_min_raise) > 1:
+            df_hatch_params = df_hatch_params.drop(df_min_raise.first_valid_index())
+
+        # Add 'Tarfet Raise' label case there is already a row with target_raise value
```
Comment (mendesfabio) [inline tech/tech.py:217]:
```suggestion
        # Add 'Target Raise' label case there is already a row with target_raise value
```

File: tech/tech.py [hunk 3]
```diff
@@ -163,92 +166,134 @@ def impact_hours_rewards(self):
...
+        # Add a new row with max_raise vale case there is no row with its value
+        if "Max Raise" not in df_hatch_params['label']:
+            impact_hour_rate = R* (self.maximum_raise / (self.maximum_raise + m*H))
+            cultural_build_tribute = (H * impact_hour_rate)/self.maximum_raise
+            df_hatch_params = df_hatch_params.append({'Total XDAI Raised': self.maximum_raise, 'Impact Hour Rate':impact_hour_rate, 'Cultural Build Tribute':cultural_build_tribute, 'Hatch tribute':hatch_tribute, 'Redeemable':hatch_tribute/(1 + cultural_build_tribute), 'label':'Max Raise'}, ignore_index=True)
+            df_hatch_params = df_hatch_params.sort_values(['Total XDAI Raised'])
+
+        df_max_raise = df_hatch_params.query("label == 'Max Raise'")
+        if len(df_max_raise) > 1:
+            df_hatch_params = df_hatch_params.drop(df_target_raise.first_valid_index())
```
Comment (mendesfabio) [inline tech/tech.py:243]:
```suggestion
            df_hatch_params = df_hatch_params.drop(df_max_raise.first_valid_index())
```
 
I'm not sure if it's wrong AHAHAH

**Review by mendesfabio — CHANGES_REQUESTED** [review]
COOL! I'm asking for some minor changes and maybe I've found a typo, @marthendalnunes!

File: tech/tech.py (hunk 3 above)
Comment (marthendalnunes) [reply tech/tech.py:243]: Yeah. It was wrong, thanks for finding it!

**Pull request merged**